### ISSUE TITLE
Increased tau phosphorylation

### ISSUE THREAD

**Issue by sarahstofel** [obophenotype/human-phenotype-ontology#7063]
**Preferred term label:**
Increased tau phosphorylation

**Synonyms**
hyperphosphorylated tau

**Definition (free text, please give PubMed ID)**
An increase in the process by which a phosphate group is introduced to the tau protein.
PMID:28625569

**Parent term (use hpo.jax.org/app)**
abnormal nervous system physiology 

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM: 104300

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0003-2307-1226

**Comment by sarahstofel** [comment]
comment: accumulation of aggregates of hyperphosphorylated tau protein are associated with many neurodegenerative disease including Alzheimer Disease, progressive supranuclear palsy, and some types of frontotemporal dementia

**Comment by pnrobinson** [comment]
@sarahstofel  -- I am wondering if this is actually a separate phenotypic feature from Neurofibrillary tangles HP:0002185, because to my knowledge the hyperphosphorylated tau is located in these tangles? Or is this something that is measure separately and affects other parts of the brain?

**Comment by pnrobinson** [comment]
@sarahstofel -- at this point I do not think this is a good candidate for a new HPO term. I am happy to reopen the issue if we can delineate a phenotypic feature that is distinct from Neurofibrillary tangles HP:0002185
